app/scripts/modules/tencentcloud/src/tencentcloud.module.ts

```diff
@@ -9,14 +9,33 @@ import { TENCENTCLOUD_SEARCH_SEARCHRESULTFORMATTER } from './search/searchResult
 import { TENCENTCLOUD_REACT_MODULE } from './reactShims/tencentcloud.react.module';
 import './validation/ApplicationNameValidator';
 
+// load pipeline stage
+import { TENCENTCLOUD_PIPELINE_STAGES_ENABLE_SG } from './pipeline/stages/enableServerGroup/enableSgStage';
```

Comment:
```suggestion
import './pipeline/stages/enableServerGroup/enableSgStage';
```
Alternately, can export all stages from a single `./pipeline/stages/index` and just import that here. (I think this is what the kubernetes provider does if you want to look at an example.)